configs/trainers/audio_jepa/bool_mask.yaml

```
@@ -0,0 +1,35 @@
+_target_: ami.trainers.bool_mask_audio_jepa_trainer.BoolMaskAudioJEPATrainer
+
+partial_dataloader:
+  _target_: torch.utils.data.DataLoader
+  _partial_: true
+  batch_size: 8
```

I-JEPAで使用されるデフォルト値が良いですね
```suggestion
  batch_size: 32
```